src/components/cards/ContentPostCarousel.tsx

```diff
@@ -40,27 +40,13 @@ export const filterContentPosts = (contentPosts: ContentPost[], countryCode?: st
  * Renders a carousel of ContentPosts using the info server data.
  */
 export const ContentPostCarousel = (props: Props) => {
-  const { countryCode } = props
+  const { contentPosts, countryCode } = props
   const { width } = useSafeAreaFrame()
   const theme = useTheme()
 
-  const [contentPosts, setContentPosts] = React.useState<ContentPost[]>([])
+  const filteredContentPosts = filterContentPosts(contentPosts, countryCode)
```

Comment:
filtering can go into the HomeScene useEffect alongside all the other code that moved